assets/javascripts/modules/cas/form.js

```diff
@@ -0,0 +1,65 @@
+define([
+    '$',
+    'bean',
+    'utils/ajax',
+    'utils/serializer'
+], function (
+    $,
+    bean,
+    ajax,
+    serializer
+) {
+    var $FORM = $('#cas-form');
+    var $RESULTS = $('dl[data-cas-result]');
+    var $ERROR = $('[data-cas-error]');
+
+    function submit() {
+        var data = serializer([].slice.call($FORM[0].elements));
+        console.log('submitting data', data);
+        ajax({
+            url: $FORM[0].action,
+            method: 'post',
+            data: data,
+            success: function(successData) {
+                console.log(successData);
```

Comment:
I want to keep it, it's only for staff and it makes things easier to debug if they have a problem